Sources/NIOPosix/BSDSocketAPIWindows.swift

```diff
@@ -701,4 +701,13 @@ extension NIOBSDSocket {
         throw ChannelError.operationUnsupported
     }
 }
+
+extension IOVector {
+    init(iov_base: UnsafeMutableRawPointer, iov_len: UInt32) {
+        self = WSABUF(
+            len: iov_len,
+            buf: iov_base.assumingMemoryBound(to: Int8.self)
```

Comment:
Can we add a documentation comment that we only use this with `ByteBuffer`-derived pointers? Otherwise the assumption here is not guaranteed to be true.